### PR TITLE
issue#2 changes in icons

### PR DESCRIPTION
**Pull request by sricharan123**
fixed issue #2

### CODE REVIEW

File: common/hipaa-password-icons.html [hunk 1]
```diff
@@ -0,0 +1,17 @@
+<link rel="import" href="../../iron-icon/iron-icon.html">
```
Comment (cherukumilli) [inline common/hipaa-password-icons.html]: @sricharan123 
You don't need this file in `common`
Pls move it to the root level.

Typically you have a common if there is something that is shared between multiple components. In this case we just have an element and nothing is shared between elements or components inside this code.

File: hipaa-password-input.html [hunk 2]
```diff
@@ -3,8 +3,8 @@
 <link rel="import" href="../paper-input/paper-input-behavior.html">
```
Comment (cherukumilli) [inline hipaa-password-input.html]: @sricharan123 
I don't see the paper-input-behavior used explicitly anywhere in your code.
I think this line should be removed.

File: hipaa-password-input.html [hunk 3]
```diff
@@ -3,8 +3,8 @@
...
 <link rel="import" href="../paper-icon-button/paper-icon-button.html">
 <link rel="import" href="../iron-icon/iron-icon.html">
-<link rel="import" href="../iron-icons/iron-icons.html">
 <link rel="import" href="../paper-input/paper-input-error.html">
+<link rel="import" href="common/hipaa-password-icons.html" async>
```
Comment (cherukumilli) [inline hipaa-password-input.html]: pls change this to:
`<link rel="import" href="hipaa-password-icons.html" async defer>`

**Comment by sricharan123** [comment]
@cherukumilli 
fixed #2 code review changes in  hipaa password input

**Review by cherukumilli — APPROVED** [review]

**Pull request merged**